.pre-commit-config.yaml

```diff
@@ -95,6 +95,11 @@ repos:
         types: [python]
         require_serial: true
         files: ^homeassistant/.+\.py$
+      - id: pylint
+        name: pylint
+        entry: pylint
+        language: system
```

Comment:
This won't work, as it needs to take the environment into account

Reply:
it works in my env (_dev container_) 🤔

Reply:
or do you mean just the `language` parameter?

Reply:
Not everbody uses a dev container. For example, I use VSCode remote over an SSH connection to a Debian 11 machine, which manages Python version and environments using pyenv and the pyenv virtualenv plugins.

And this is not the only variant 😉

Reply:
> And this is not the only variant 😉

yeah, I know ... there are many different preferences out there to get things done 😆